pkg/cleanup/cleanup.go

```diff
@@ -0,0 +1,52 @@
+package cleanup
+
+import (
+	"github.com/containous/maesh/pkg/k8s"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Cleaner is an interface for the cleanup methods.
+type Cleaner interface {
+	CleanShadowServices() error
+}
+
+// Ensure the Prepare fits the Preparer interface
```

Comment:
Copy paste engineer! :p 

```suggestion
// Ensure the Cleanup fits the Cleaner interface
```

Reply:
Only the best.